### PR TITLE
allow using influxdb existing secret in chronograf

### PR DESCRIPTION
**Pull request by jetersen**

### CODE REVIEW

File: charts/chronograf/values.yaml [hunk 1]
```diff
@@ -131,3 +131,10 @@ envFromSecret: ""
 # - name: chronograf-output-influxdb2
 #   mountPath: /var/lib/chronograf/conf.d
 #   subPath: influxdb2.conf
+
+influxdb:
+  ## Name of an existing Secrect used to set the environment variables for the
+  ## InfluxDB user and password. The expected keys in the secret are
+  ## `influxdb-user` and `influxdb-password`.
+  ##
+  # existingSecret: influxdb-auth
```
Comment (jetersen) [inline charts/chronograf/values.yaml:140]: I think calling just `existingSecret` as in the kapacitor would be confusing.

https://github.com/influxdata/helm-charts/blob/20aeb75f75ecf739c8134edec7c5e8ff249d85f5/charts/kapacitor/values.yaml#L68

**Review by rawkode — APPROVED** [review]

**Review by rawkode — DISMISSED** [review]

**Review by timhallinflux — APPROVED** [review]

**Comment by timhallinflux** [comment]
@jetersen can you look at the merge conflict?

**Comment by jetersen** [comment]
@timhallinflux done 👍

**Comment by timhallinflux** [comment]
@jetersen looks like you need to bump the chart version as well.

**Comment by jetersen** [comment]
@timhallinflux done 😓

**Comment by timhallinflux** [comment]
thanks @jetersen !

**Pull request merged**